cfn-resources/teams/test/README.md

```diff
@@ -35,4 +35,25 @@ The Atlas organization must be manually provided.
 
 ## Important Links
 - [API Documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Teams)
-- [Resource Usage Documentation](https://www.mongodb.com/docs/atlas/access/manage-teams-in-orgs/)
\ No newline at end of file
+- [Resource Usage Documentation](https://www.mongodb.com/docs/atlas/access/manage-teams-in-orgs/)
+
+## Unit Testing Locally
```

Comment:
I think this file should contain a reference to [TESTING.md.md](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/TESTING.md.md) and then contain the acceptance criteria for the Manual QA similar to [cluster/test/README.md](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/cluster/test/README.md)